openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,10 +281,15 @@ The following additional considerations are given for pre-existing Authorization
 : OPTIONAL. Defined in [@!RFC6749]. The Wallet MAY allow Verifiers to request presentation of Verifiable Credentials by utilizing a pre-defined scope value. See (#request_scope) for more details.
 
 `response_mode`:
-: OPTIONAL. Defined in [@!OAuth.Responses]. This parameter is used (through the new Response Mode `direct_post`) to ask the Wallet to send the response to the Verifier via an HTTPS connection (see (#response_mode_post) for more details). It is also used to request signing and encrypting (see (#jarm) for more details). If the parameter is not present, the default value is `fragment`. 
+: OPTIONAL. Defined in [@!OAuth.Responses]. This parameter is used (through the new Response Mode `direct_post`) to ask the Wallet to send the response to the Verifier via an HTTPS connection (see (#response_mode_post) for more details). It is also used to request signing and encrypting (see (#jarm) for more details). If the parameter is not present, the default value is `fragment`.
 
-The following is a non-normative example of an Authorization Request: 
+The Verifier MAY send an Authorization Request using either of these 3 options:
+1. Passing as URL with encoded parameters
+2. Passing a request object as value
+3. Passing a request object by reference
+As defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].
```

Comment:
```suggestion
2 and 3 are defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].
```

it's only 2 and 3 that are defined in JAR, right?

Reply:
good catch! you are right. `presentation_definition` isn't defined in RFC9101